packages/cubejs-databricks-jdbc-driver/src/DatabricksDriver.ts

```diff
@@ -657,10 +684,16 @@ export class DatabricksDriver extends JDBCDriver {
       pathname.split(`${this.config.exportBucket}/`)[1];
     const expr = new RegExp(`${foldername}\\/.*\\.csv$`, 'i');
 
-    const credential = new StorageSharedKeyCredential(
-      account,
-      this.config.azureKey as string,
-    );
+    const credential = this.config.azureKey
```

Comment:
Should still default to `StorageSharedKeyCredential`

Reply:
Fixed here. Thanks.